n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -261,6 +271,20 @@ def check_positive(value):
 
         parser.add_argument("--hyperopt", help="Enable hyperopt scan", default=None, type=int)
         parser.add_argument("--restart", help="Enable hyperopt restarts", action="store_true")
+        parser.add_argument(
+            "--parallel-hyperopt",
+            help="Enable hyperopt run in parallel with MongoDB",
+            action="store_true",
+        )
```

Comment:
I should have asked this already in the other PR but what should be the intended behavior if some random user sets both `--restart --parallel-hyperopt`?